ui/src/app/settings/utils.js

```diff
@@ -20,16 +20,14 @@ const simpleObjectEquality = (obj1, obj2) => {
  * @returns {Boolean} Form is disabled.
  */
 const formikFormDisabled = formikProps => {
-  const { initialValues, isSubmitting, errors, values } = formikProps;
+  const { initialValues, errors, values } = formikProps;
   let hasErrors = false;
 
   if (errors) {
     hasErrors = Object.keys(errors).length > 0;
   }
 
-  return (
-    isSubmitting || simpleObjectEquality(initialValues, values) || hasErrors
```

Comment:
isSubmitting causes issues when controlling the state from redux (e.g. when server errors are returned in the state).

Reply:
That's fair. Might be a good idea to disable the button if `saving` is true in that case.

Reply:
Is there a way to disable the button without the vanilla styles changing? Maybe it's OK for them to change, would just look different to the designs that's all.

Reply:
Nevermind, looks like that case is already handled in the action button.